packages/client/server.js

```diff
@@ -1,36 +1,36 @@
-const path = require('path');
-const fs = require('fs');
-const express = require('express');
-const expressStaticGzip = require('express-static-gzip');
-const packageRoot = require('app-root-path').path;
-const https = require('https');
-const http = require('http');
+import { join } from 'path';
```

Comment:
All of these need to go back to using require() instead of import, as this is a .js file and is run as plain javascript, not TypeScript.